buddyboss/buddyboss-platform
# #531 - Added buddypanel menu fix for group, forums, photos when WPML is active

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #531 .

### How to test the changes in this Pull Request:

1. Activate WPML, BuddyPress Multilingual plugins together with BuddyBoss Platform
2. In WPML > Languages, make sure that Language URL format is set to Different languages in directories.
3. Make sure that you have already set menu links on both BuddyPanel and titlebar
4. On the frontend, go to the profile page and navigate on groups and forums

### Proof Screenshots or Video
https://www.loom.com/share/85bf4895e7da44c7b59407f5e985a236


### Other information:




### Changelog entry

>  Added buddypanel menu fix for group, forums, photos when WPML is active
